cfme/cloud/tenant.py

```diff
@@ -254,6 +306,8 @@ def wait_for_appear(self, timeout=600):
                         message='Wait for cloud tenant to appear', fail_func=self.browser.refresh)
 
     def update(self, updates, wait=True):
+        if not self.exists:
```

Comment:
Removed it, and in `create` I made the `wait=True` by default, because tenants take some time to appear.